deviceModeInit.js

```diff
@@ -467,6 +467,36 @@ var rudderTracking = (function () {
       });
   }
 
+  /**
+   * Checks whether the url for the product contains the variant id
+   * @returns {string} variant id or null
+   */
+  function findVariantIdInURL() {
+    const matches = window.location.href.match(/\d{8,20}/);
+    if(matches) {
+      return matches[0]
+    }
+    return null;
+  };
+
+  /**
+   * Returns the sku value for the variant matching the id in url 
+   * @param {*} payload 
```

Comment:
Give description of the paylaod what it means?

Reply:
Added